static/js/transform-facets.js

```diff
@@ -12,33 +14,45 @@ export default function transformFacets (facets, config) {
   }
 
   return facets.map(facet => {
-    const isConfigurationForFacet = facet.fieldId in config.fields;
-    if (!isConfigurationForFacet) {
+    const hasConfigurationForFacet = facet.fieldId in config.fields;
+    if (!hasConfigurationForFacet) {
       return facet;
     }
     const facetConfig = config.fields[facet.fieldId];
 
     let options = facet.options;
 
     if ('fieldLabels' in facetConfig) {
-      options = facet.options.map(option => {
-        const fieldLabels = facetConfig.fieldLabels;
+      const fieldLabels = facetConfig.fieldLabels;
 
+      options = facet.options.map(option => {
         const displayName = (option.displayName in fieldLabels)
           ? fieldLabels[option.displayName]
           : option.displayName;
-
-        return Object.assign({}, option, { displayName });
+        return { ...option, displayName };
       })
     }
 
-    const filterOptionsConfig = Object.entries(facetConfig).reduce((filterOptions, [option, value]) => {
-      if (option !== 'fieldLabels') {
-        filterOptions[option] = value;
+    if ('optionsOrder' in facetConfig) {
+      const { optionsOrder } = facetConfig;
+      if (optionsOrder === 'ASC') {
+        options = options.sort((a, b) =>  a.displayName.toString().localeCompare(b.displayName.toString()));
+      } else if (optionsOrder === 'DESC') {
+        options = options.sort((a, b) =>  b.displayName.toString().localeCompare(a.displayName.toString()));
+      } else {
+        console.error(`Unknown facet optionsOrder "${optionsOrder}" for the "${facet.fieldId}" facet.`);
       }
-      return filterOptions;
-    }, {});
+    }
+
+    // Don't expose Hitchhiker theme specific details to the SDK
+    const filterOptionsConfig = clonedeep(facetConfig);
```

Comment:
Do we need to do a deep copy of it? I think we can use spread operator here like `const { fieldLabels, optionsOrder, ...filterOptionsConfig } = facetConfig;` and just pass in filterOptionsConfig that way

Reply:
ah, yeah I was just following the original code's logic, but that's better!